docs/reference/plugins/py-terminal.md

```diff
@@ -4,19 +4,35 @@ This is one of the core plugins in PyScript, which is active by default. With it
 
 ## Configuration
 
-You can control how `<py-terminal>` behaves by setting the value of the  `terminal` configuration in your `<py-config>`.
+You can control how `<py-terminal>` behaves by setting the value of the `terminal` configuration in your `<py-config>`, together with the `docked` one.
+
+For the **terminal** field, these are the values:
 
 | value | description |
 |-------|-------------|
 | `false` | Don't add `<py-terminal>` to the page |
 | `true` | Automatically add a `<py-terminal>` to the page |
 | `"auto"` | This is the default. Automatically add a `<py-terminal auto>`, to the page. The terminal is initially hidden and automatically shown as soon as something writes to `stdout` and/or `stderr` |
 
+For the **docked** field, these are the values:
+
+| value | description |
+|-------|-------------|
+| `false` | Don't dock `<py-terminal>` to the page |
+| `true` | Automatically dock a `<py-terminal>` to the page |
+| `"docked"` | This is the default. Automatically add a `<py-terminal docked>`, to the page. The terminal, once visible, is automatically shown at the bottom of the page, covering the width of such page |
```

Comment:
I think the `docked` option is just another way to place the terminal in the page right? Perhaps we should just add the "docked" part to the above table, what do you think?

Reply:
the previous table is about `terminal` field, and `docked` is not a property of the *terminal* field, it's a property a part: `<py-terminal auto docked></py-terminal>` as example, so previous table would be probably confusing, as that's about the `terminal = ...` value? 🤔